src/ucp/api/ucp.h

```diff
@@ -741,15 +741,19 @@ typedef enum {
                                                         which require
                                                         synchronization with the
                                                         remote peer before releasing
-                                                        the local send buffer */
+                                                        the local send buffer.
+                                                        This flag and UCP_OP_ATTR_FLAG_MULTI_SEND
+                                                        are mutually exclusive. */
     UCP_OP_ATTR_FLAG_FORCE_IMM_CMPL = UCS_BIT(18), /**< force immediate complete
                                                         operation, fail if the
                                                         operation cannot be
                                                         completed immediately */
     UCP_OP_ATTR_FLAG_MULTI_SEND     = UCS_BIT(19)  /**< optimize for bandwidth of
                                                         multiple in-flight operations,
                                                         rather than for the latency
-                                                        of a single operation */
+                                                        of a single operation.
+                                                        This flag and UCP_OP_ATTR_FLAG_FAST_CMPL
+                                                        are mutually exclusive. */
```

Comment:
Can document it in the description of `ucp_op_attr_t` instead for each flag.
WDYT?

Reply:
add it to MULTI_SEND only. No need to duplicate.